aiida/orm/implementation/django/group.py

```diff
@@ -117,14 +117,13 @@ def is_stored(self):
 
     def store(self):
         if not self.is_stored:
-            sid = transaction.savepoint()
-            try:
-                self.dbgroup.save()
-            except IntegrityError:
-                transaction.savepoint_rollback(sid)
-                raise UniquenessError("A group with the same name (and of the "
-                                      "same type) already "
-                                      "exists, unable to store")
+            with transaction.atomic():
```

Comment:
The try/except should be outside of the with statement, as shown in the example on the docs:
https://docs.djangoproject.com/en/1.7/topics/db/transactions/#django.db.transaction.atomic